app/schemas/schema.js

```diff
@@ -1,112 +1,230 @@
 const schema = {
-  "title": "Request to Participate",
   "type": "object",
   "required": [
-    "organization",
-    "streetNumber",
-    "streetName",
-    "city",
-    "postalCode"
+    // "Organization",
+    // "Registration Number",
+    // "Street Number",
+    // "Street Name",
+    // "City",
+    // "Postal Code",
+    // "Primary Contact",
+    // "Postion",
+    // "Contact Email",
+    // "Telephone",
+    // "Zone Feedback",
+    // "Wired Broadband",
+    // "Fixed Wireless",
+    // "Last Mile Support",
+    // "Anticipated Projects",
+    // "Underserved Areas",
+    // "Satellite Proposal",
+    // "Capital Cost Funding",
+    // "Capital Cost Explanation",
+    // "Last Mile Broadband",
+    // "Last Mile Fixed Wireless",
+    // "Last Mile New Backbone",
+    // "Highway Cellular"
   ],
   "properties": {
-    "Applicant Profile:": {
+    "Organization Profile": {
+      "title": "Organization Profile",
+      "type": "object",
       "properties": {
-        "organization": {
-          "title": "Applicant organization (legal name):",
+        "Organization": {
+          "title": "Organization name (legal name)",
           "type": "string"
         },
-        "registrationNumber": {
+        "Registration Number": {
           "type": "string",
-          "title": "Band number, society number, or business registration number (if applicable):"
+          "title": "Band number, society number, or business registration number (optional)"
         },
-        "streetNumber": {
+        "Unit Number": {
           "type": "number",
-          "title": "Street number:"
+          "title": "Unit number (optional)"
```

Comment:
It makes sense to have the `(optional)` in the title for now, but let's get a ticket opened upstream? If a field isn't marked required then the toolkit should automatically add this label.

Reply:
hmm have not seen any optional labels applied automatically.

Reply:
https://github.com/button-inc/service-development-toolkit/issues/379